src/main/java/net/juniper/netconf/Device.java

```diff
@@ -189,8 +187,6 @@ private String createHelloRPC(List<String> capabilities) {
      */
     private NetconfSession createNetconfSession() throws NetconfException {
         if (!isConnected()) {
```

Comment:
it looks like we don't do anything interesting in this method anymore, now that you in line 138 do the 
`this.sshClient = (sshClient != null) ? sshClient : new JSch();`

should we make the createNeconfSession a no-op and deprecate it?

Reply:
We use `createNetconfSession` in `connect()` method and sshClient initialization is a different thing than session creation. no-op means no-operations, correct?

Reply:
ah ha! when i was looking at the github ui, it looked like the create netconf session method code ended below  line 192.

got it!. 👍

Reply:
oh, yeah. GitHub sometimes is very confusing